README.md

```diff
@@ -113,6 +113,11 @@ Configuration for the different connectors. If you use a connector that requires
 
 See the [web3-react documentation](https://github.com/NoahZinsmeister/web3-react/tree/v6/docs) for more details.
 
+#### ethereum
+
+An [EIP-1193](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1193.md) compatible provider
```

Comment:
Nitpick, we generally don't add new lines in README files for same-paragraph text.

Reply:
Gotcha; Here I'm really just following this document's style, as it's done on the other parts of it. Should we update the whole thing? 😄

Reply:
Hmm for the most part I think it doesn't break lines, but please do update any parts that don't follow this!